<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AzureSpeakerGUIClass</class>
 <widget class="QMainWindow" name="AzureSpeakerGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AzureSpeakerGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="Welcome">
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>欢迎使用！</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="Start">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>开始朗读</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="Stop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>停止</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QWidget" name="widget_3" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="SelectFile">
         <property name="text">
          <string>朗读文本：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="SelectLanguage">
         <property name="text">
          <string>语言：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="FilePath">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="VoiceName"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="SelectVoice">
         <property name="text">
          <string>语音：</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="VoiceLanguage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="Reload">
         <property name="text">
          <string>重新加载文件内容</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="SelectButton">
         <property name="text">
          <string>选择</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AzureSpeakerGUI.qrc"/>
 </resources>
 <connections/>
</ui>
